web-marketplace/src/components/organisms/Order/Order.tsx

```diff
@@ -0,0 +1,144 @@
+import { msg, Trans } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import {
+  Avatar,
+  CardContent,
+  CardHeader,
+  CardMedia,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material';
+
+import OutlinedButton from 'web-components/src/components/buttons/OutlinedButton';
+import Card from 'web-components/src/components/cards/Card';
+import CertifiedDocumentIcon from 'web-components/src/components/icons/CertifiedDocumentIcon';
+// import ReceiptIcon from 'web-components/src/components/icons/ReceiptIcon';
+import { Image } from 'web-components/src/components/image';
+import ProjectPlaceInfo from 'web-components/src/components/place/ProjectPlaceInfo';
+import { PrefinanceTag } from 'web-components/src/components/PrefinanceTag/PrefinanceTag';
+import { Title } from 'web-components/src/components/typography';
+
+import { getProjectCardBodyTextMapping } from 'lib/constants/shared.constants';
+
+import { ORDER_STATUS } from './Order.constants';
+import { OrderLabel } from './Order.Label';
+import { OrderSummary } from './Order.Summary';
+import { OrderDataProps, RetirementInfoData } from './Order.types';
+
+export const Order = (orderData: OrderDataProps) => {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+  const { _ } = useLingui();
+
+  const { project } = orderData;
+  const { retirementInfo, blockchainDetails, credits, paymentInfo, status } =
+    orderData.order;
+  const isRetirementTradable =
+    (retirementInfo.data as RetirementInfoData).tradableCredits !== null;
+  return (
+    <Card className="bg-grey-100 border-[1px] border-grey-200">
+      {isMobile && (
```

Comment:
I think is better to not render them instead of just hide them as hidden elements still exist in the DOM. Also, we use `useMediaQuery` in many places already and thought we'll probably end up needing to create a `useMediaQuery` hook if we migrate from MUI.